apps/explorer/src/components/OwnedObjects/index.tsx

```diff
@@ -138,7 +138,12 @@ export function OwnedObjects({ id }: { id: string }) {
 	}
 
 	return (
-		<div className={clsx(!noAssets && 'h-coinsAndAssetsContainer md:h-full')}>
+		<div
+			className={clsx(
+				!noAssets && 'h-coinsAndAssetsContainer',
+				!noAssets && fullHeight && 'h-full',
```

Comment:
Why not keep the full height through `md:h-full`? It seems like we're doing roughly the same thing below. Also if you set `fullHeight`, then both `h-coinsAndAssetsContainer` and `h-full` will be set which seems wrong.